client/src/components/client-page/index.jsx

```diff
@@ -0,0 +1,846 @@
+  };
+
+  const handleStatusExitChange = (event) => {
+    setEditedStatusExit(event.target.value);
+  };
+
+  const handleStatus3Change = (event) => {
+    setEditedStatus3(event.target.value);
+  };
+
+  const handleStatus6Change = (event) => {
+    setEditedStatus6(event.target.value);
+  };
+
+  const handleStatus9Change = (event) => {
+    setEditedStatus9(event.target.value);
+  };
+
+  const handleStatus12Change = (event) => {
+    setEditedStatus12(event.target.value);
+  };
+
+
+  const handleChipDelete = (chipType) => {
+    switch (chipType) {
+      case 'status':
+        setEditedStatus('');
+        break;
+      case 'statusExit':
+        setEditedStatusExit('');
+        break;
+      case 'status3':
+        setEditedStatus3('');
+        break;
+      case 'status6':
+        setEditedStatus6('');
+        break;
+      case 'status9':
+        setEditedStatus9('');
+        break;
+      case 'status12':
+        setEditedStatus12('');
+        break;
+      default:
+        break;
+    }
+  };
+
+  const handleSave = () => {
+    
+    const updatedClientInfo = {
+      ...clientInfo,
+      firstName: editedName,
+      email: editedEmail,
+      phone: editedPhone,
+      status: editedStatus,
+      status_at_exit: editedStatusExit,
+      status_at_3: editedStatus3,
+      status_at_6: editedStatus6,
+      status_at_9: editedStatus9,
+      status_at_12: editedStatus12,
+    };
+
+    
+    onSaveChanges(updatedClientInfo);
+    setIsEditMode(!isEditMode);
+  };
+
+  const isStatusExitVisible = editedStatus === 'Closed';
+
+  return (
+    <div style={{ marginLeft: "40px", marginRight: "40px", marginTop: "40px" }}>
+      <div
+        style={{ display: "flex", alignItems: "center", marginBottom: "10px" }}
+      >
+        <IconButton onClick={handleBackClick} 
+        sx={{ color: "gray", marginRight: 2, marginLeft: 2 }}>
+        <ArrowBackIcon/>
+        </IconButton>
+        <Box className="title">
+          <Typography marginBottom={1} variant="h4" align="left">
+            {clientInfo.firstName}
+          </Typography>
+          <Typography variant="subtitle2" align="left">
+            Client
+          </Typography>
+        </Box>
+              <Box
+            sx={{
+              marginLeft: "auto",
+              display: "flex",
+              alignItems: "center",
+              borderRadius: "8px",
+              boxShadow: 2,
+              p: 3,
+            }}
+          >
+            <Box sx={{ textAlign: "center", mr: 2 }}>
+              <Typography variant="subtitle2" sx={{ mb: 1 }}>
+                Owner
+              </Typography>
+              <UserChipComponent user={owner} onClick={handleClick} edit />
+            </Box>
+            <Box sx={{ textAlign: "center" }}>
+              <Typography variant="subtitle2" sx={{ mb: 1 }}>
+                Creator
+              </Typography>
+              <UserChipComponent user={creator} />
+            </Box>
+          </Box>
```

Comment:
We need to add the change owner dialog component here. Even though it will not do anything yet, it will give us the functionality to link it up afterwards and not forget

Component -> https://github.com/uoftblueprint/tces/blob/main/client/src/components/shared/change-owner-dialog/index.jsx

Implementation -> https://github.com/uoftblueprint/tces/blob/main/client/src/components/edit-job-lead-component/edit-job-lead-header/index.jsx

And then just add a comment that the necessary updates to the component need to be made when routing